README.md

```diff
@@ -4,12 +4,16 @@ This module is used to add share tools to VJ content.
 
 ##Example use
 
+###Requirements
+Sharetools has a jquery dependency. Make sure 'jquery' is findable in your RequireJS paths.
```

Comment:
This is a bit strange for a bower module, could we not list it as a dependency on the package?

Also jquery is called jquery-1 / jquery-2 inside the IDT. We could maybe code something to check for jquery-2, jquery-1 and jquery and then return whichever it finds? Or failing that maybe look for window.jQuery